sources/api/netdog/Cargo.toml

```diff
@@ -9,7 +9,7 @@ publish = false
 exclude = ["README.md"]
 
 [features]
-default = ["systemd-networkd"]
+default = ["wicked"]
 wicked = []
```

Comment:
Are the unit tests something we can fix here? I think advocating for `systemd-networkd` as the default makes sense.

Reply:
will fix